src/Features/Core/Portable/Completion/Providers/ImportCompletionProvider/AbstractTypeImportCompletionService.CacheEntry.cs

```diff
@@ -87,7 +87,7 @@ static CompletionItem GetAppropriateAttributeItem(CompletionItem attributeItem,
                     if (attributeItem.DisplayText.TryGetWithoutAttributeSuffix(isCaseSensitive: isCaseSensitive, out var attributeNameWithoutSuffix))
                     {
                         // We don't want to cache this item.
-                        return ImportCompletionItem.CreateAttributeItemWithoutSuffix(attributeItem, attributeNameWithoutSuffix);
+                        return ImportCompletionItem.CreateAttributeItemWithoutSuffix(attributeItem, attributeNameWithoutSuffix, CompletionItemFlags.Expanded);
```

Comment:
Do we need to add this flag in other places as well? Where did the "attributeItem" come from originally?

Reply:
Or should this be passing `attributeItem.Flags | CompletionItemFlags.Expanded` if the existing item also had another flag?